instantsearch/src/main/java/com/algolia/instantsearch/helpers/Searcher.java

```diff
@@ -316,7 +319,8 @@ public void requestCompleted(@NonNull JSONObject content, @Nullable AlgoliaExcep
                     } else {
                         endReached = true;
                     }
-                    EventBus.getDefault().post(new ResultEvent(content, query, currentRequestId));
+                    SearchResults results = new SearchResults(content, hierarchicalFacets);
```

Comment:
Might as well inline `results` in the next line

Reply:
Put it inline will make the next line too long.